### PR TITLE
add rajeshpg bio

### PR DESCRIPTION
**Pull request by rajeshpg**

### CODE REVIEW

File: mentors/scala/r.json [hunk 1]
```diff
@@ -14,5 +14,13 @@
...
-  }
+  },
+  {
+      "github_username": "rajeshpg",
+      "name": "Rajesh Pitty",
+      "link_text": null,
+      "link_url": null,
+      "avatar_url": null,
+      "bio": "a passionately curious scala functional programmer"
+    }
```
Comment (kotp) [inline mentors/scala/r.json]: Indentation should be exactly two spaces.

File: mentors/scala/r.json [hunk 2]
```diff
@@ -14,5 +14,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I really like working in Scala.  It allows me to come up with elegant, functional code that is also concise and type safe."
-  }
+  },
+  {
+      "github_username": "rajeshpg",
```
Comment (kotp) [inline mentors/scala/r.json]: From here for indentation

**Comment by rpottsoh** [comment]
@rajeshpg so you have any questions or need some help with making the requested changes?

File: mentors/scala/r.json (hunk 1 above)
Comment (rajeshpg) [reply mentors/scala/r.json]: fixed the indentation.

File: mentors/scala/r.json (hunk 2 above)
Comment (rajeshpg) [reply mentors/scala/r.json]: fixed the indentation

**Pull request merged**